content/articles/2017-08-tidyr-0.7.0.Rmd

```diff
@@ -0,0 +1,214 @@
+---
+title: tidyr 0.7.0
+slug: tidyr-0.7.0
+description: >
+  The next installment of tidyr is finally on CRAN! This version
+  brings tidy eval to a crucial component of the tidyverse workflow.
+author: Lionel Henry
+date: 2018-08-15
+photo:
+  url: https://unsplash.com/photos/8tem2WpFPhM
+  author: Radek Grzybowski
+categories: [packages]
+---
+
+```{r setup, include = FALSE}
+library("tidyr")
+library("dplyr")
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We are happy to announce that tidyr 0.7.0 is now available on
+CRAN. This update makes tidyr compatible with the tidy evaluation
+features introduced in dplyr 0.7.0. It also uses the new tidyselect
```

Comment:
I'd add some context like: This version is mostly about ensuring consistent semantics across the tidyverse. It also brings tidy eval to tidyr, make it easier to program with tidy (and link to dplyr vignette?).